include/matx/generators/random.h

```diff
@@ -401,6 +401,11 @@ template <typename T, int RANK> class randomTensorView_t {
         using scalar_type = T;
         using matxop = bool;
 
+        static_assert(std::is_same_v<T, float> || 
```

Comment:
We should update the documentation for random to reflect it only works for float types. currently we don't have any verbiage to outline that